### PR TITLE
Add build action step in action

### PR DESCRIPTION
**Pull request by bigint**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -12,13 +12,17 @@ jobs:
...
+      run: yarn
+
     - name: Lint with ES Lint
-      run: |
-        yarn
-        yarn lint
+      run: yarn lint
+
+    - name: Build
+      run: yarn
```
Comment (filiptronicek) [inline .github/workflows/ci.yml:28]: Maybe you meant `yarn build`?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -12,13 +12,17 @@ jobs:
...
+    - name: Install Dependencies
+      run: yarn
+
     - name: Lint with ES Lint
-      run: |
-        yarn
-        yarn lint
+      run: yarn lint
+
+    - name: Build
```
Comment (filiptronicek) [inline .github/workflows/ci.yml:27]: Won't there be any issues with Prisma and environment variables? I think I had these issues over at https://github.com/filiptronicek/CountDowner.

**Review by filiptronicek — CHANGES_REQUESTED** [review]
Thanks for the contrib, @bigint, had some thoughts 😄❤️

File: .github/workflows/ci.yml (hunk 1 above)
Comment (bigint) [reply .github/workflows/ci.yml:28]: oops typo 🤣 forgot to type

File: .github/workflows/ci.yml (hunk 2 above)
Comment (bigint) [reply .github/workflows/ci.yml:27]: What kind of issue?

Comment (filiptronicek) [reply .github/workflows/ci.yml:27]: Well there is no database, so wouldn't it have a problem generating the client when it is not accessible?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (bigint) [reply .github/workflows/ci.yml:28]:
```suggestion
      run: yarn build
```

File: .github/workflows/ci.yml (hunk 2 above)
Comment (bigint) [reply .github/workflows/ci.yml:27]: Ooo wait just updated the step to yarn build, if failed we can set env vars in gh secrets 🙇🏻

Comment (bigint) [reply .github/workflows/ci.yml:27]: Oops CI fails but it is unrelated to DB, its a problem with another breaking change in Next.js 😭

Comment (filiptronicek) [reply .github/workflows/ci.yml:27]: Wonders, working now

**Review by filiptronicek — APPROVED** [review]
LGTM

**Pull request merged**